virtual/types/types.go

```diff
@@ -43,4 +43,6 @@ type ActorReferencePhysical interface {
 	ServerID() string
 	// The address of the referenced actor.
 	Address() string
+	// server version is increased after any missed heartbeat
+	ServerVersion() int64
```

Comment:
`ServerVersion is increased everytime a server's heartbeat expires and then resumes. It helps ensure that a server can always identify when there has been a window of time during which it was considered inactive/dead for correctness purposes.